plugin/trino-kafka-event-listener/src/main/java/io/trino/plugin/eventlistener/kafka/KafkaEventListenerConfig.java

```diff
@@ -0,0 +1,211 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.eventlistener.kafka;
+
+import com.google.common.base.Splitter;
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+import jakarta.validation.constraints.AssertTrue;
+import jakarta.validation.constraints.NotEmpty;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+public class KafkaEventListenerConfig
+{
+    private static final Splitter LIST_SPLITTER = Splitter.on(",").trimResults().omitEmptyStrings();
+    private static final Splitter.MapSplitter MAP_SPLITTER = LIST_SPLITTER.withKeyValueSeparator("=");
+
+    private boolean anonymizationEnabled;
+    private boolean publishCreatedEvent = true;
+    private boolean publishCompletedEvent = true;
+    private Optional<String> completedTopicName = Optional.empty();
+    private Optional<String> createdTopicName = Optional.empty();
+    @NotEmpty
```

Comment:
We put these at getter by convention. Also you can leave it null.

Same for `brokerEndpoints` (I also think it has to have `@NonEmpty` annotation)